sotodlib/site_pipeline/make_relcal_model.py

```diff
@@ -0,0 +1,372 @@
+'''This module receives the fitted detector optical response 
+from the analyze-bright-ptsrc after the detmap has been applied, 
+and determines a 'flat field' for the detectors. The script can 
+take one single scan or multiple scans. The final output is a 
+set of per-detector calibration factors of order 1 associated 
+with det_id, to be multiplied to the detector response. 
+'''
+import sys
+import numpy as np
+import os
+import yaml
+import h5py
+from argparse import ArgumentParser
+import matplotlib.pyplot as plt
+from sotodlib.core import Context, metadata
+import sotodlib.io.metadata as io_meta
+from sotodlib.site_pipeline import util
+
+def get_parser():
+
+    parser = ArgumentParser()
+
+    parser.add_argument('config_file', help=
+        'Configuration file.')
+ 
+    parser.add_argument('--obs_id', nargs='*', help=
+         'Observations to summarize over.')
+
+    parser.add_argument('--obs_id_file', help=
+        'Path to a txt that stores obs_ids.')
+
+    parser.add_argument('-w', '--wafers', help=
+         'The wafer or wafers to run model on.')
+
+    parser.add_argument('-q', '--obs_query', help = 
+        'String to pass to ctx.obsdb.query()')
+
+    parser.add_argument('-v', '--verbose', action= 'count',
+        default = 0, help ='Pass multiple times to increase.')
+
+    parser.add_argument('--make_plot', action = 'store_true',
+        help = 'Plot response over time.')
+
+    parser.add_argument('--save_plot', action = 'store_true',
+        help = 'Save plots to output directory.')
+
```

Comment:
My picky preferences again, sorry ... can you replace these with:
```
--show (show the plots
--no-plots (don't make or save the plots)
```

Because I want the default behavior to be to save the plots.